channels/alpha

```diff
@@ -78,6 +78,10 @@ spec:
       providerID: gce
       architectureID: amd64
       kubernetesVersion: ">=1.18.0"
+    - name: "Canonical:UbuntuServer:20.04-LTS:latest"
+      providerID: azure
+      architectureID: amd64
+      kubernetesVersion: ">=1.18.0"
```

Comment:
Any special reason for 1.18+? I would use the same version as when kOps started supporting Azure.
```suggestion
      kubernetesVersion: ">=1.20.0"
```

Reply:
I see. No strong reason. Let me update this and the other channel.

Reply:
Great. Thanks. :)